.adversarial/inputs/AEL-0013-code-review-input.md

```
@@ -0,0 +1,61 @@
+# Code Review: AEL-0013 — Migrate to .kit/ Directory Layout
```

markdownlint-cli2 (0.22.0)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)